packages/webpack-config-single-spa/package.json

```diff
@@ -25,5 +25,9 @@
     "clean-webpack-plugin": "^3.0.0",
     "unused-files-webpack-plugin": "^3.4.0",
     "webpack-bundle-analyzer": "^3.6.0"
+  },
+  "optionalDependencies": {
```

Comment:
These have to be listed as optionalDependencies in order for them to be resolved correctly